llvm/test/CodeGen/SPIRV/hlsl-intrinsics/SV_GroupIndex.ll

```diff
@@ -1,9 +1,6 @@
 ; RUN: llc -O0 -verify-machineinstrs -mtriple=spirv-vulkan-unknown %s -o - | FileCheck %s
 ; RUN: %if spirv-tools %{ llc -O0 -mtriple=spirv-vulkan-unknown %s -o - -filetype=obj | spirv-val --target-env vulkan1.3 %}
```

Comment:
Chris Bieneman raised the point that we shouldn't have a target environment (vulkan1.3) that is different than the target triple (spirv-vulkan-unknown).